src/plugins/ResizeReorder/ResizeCell.js

```diff
@@ -25,24 +25,22 @@ class ResizeCell extends React.PureComponent {
       minWidth,
       maxWidth,
       onColumnResizeEnd,
-      onColumnReorderEnd,
       rowIndex,
-      left,
       touchEnabled,
       cellGroupType,
       onColumnReorderStart,
       ...props
     } = this.props;
 
     let style = {
-      height: props.height,
-      width: props.width - BORDER_WIDTH,
+      height: this.props.height,
```

Comment:
we have already defined `props` you don't need to use this.props here and in next lines.

Reply:
corrected